src/app/legacy/containers/ImageWithPlaceholder/index.test.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import { render, waitFor } from '@testing-library/react';
-import { shouldMatchSnapshot } from '#psammead/psammead-test-helpers/src';
-import { C_GHOST } from '#psammead/psammead-styles/src/colours';
+import { waitFor } from '@testing-library/react';
+import { GHOST } from '../../../components/ThemeProvider/palette';
```

Comment:
Should I refactor this test to not use an imported colour value?

Reply:
I would say leave it, not too concerning for a test